Index: src/main/java/com/example/casestudy/HelloServlet.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.casestudy;\r\n\r\nimport java.io.*;\r\nimport javax.servlet.http.*;\r\nimport javax.servlet.annotation.*;\r\n\r\n@WebServlet(name = \"helloServlet\", value = \"/hello-servlet\")\r\npublic class HelloServlet extends HttpServlet {\r\n    private String message;\r\n    ajsdflkjas;f\r\n\r\n\r\n    public void init() {\r\n        message = \"Hello World!\";\r\n    }\r\n\r\n    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {\r\n        response.setContentType(\"text/html\");\r\n\r\n        // Hello\r\n        PrintWriter out = response.getWriter();\r\n        out.println(\"<html><body>\");\r\n        out.println(\"<h1>\" + message + \"</h1>\");\r\n        out.println(\"</body></html>\");\r\n    }\r\n\r\n    public void destroy() {\r\n    }\r\n}
===================================================================
diff --git a/src/main/java/com/example/casestudy/HelloServlet.java b/src/main/java/com/example/casestudy/HelloServlet.java
--- a/src/main/java/com/example/casestudy/HelloServlet.java	
+++ b/src/main/java/com/example/casestudy/HelloServlet.java	
@@ -7,7 +7,7 @@
 @WebServlet(name = "helloServlet", value = "/hello-servlet")
 public class HelloServlet extends HttpServlet {
     private String message;
-    ajsdflkjas;f
+
 
 
     public void init() {
